XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -492,100 +609,56 @@ private Vector3 GetRotationAxis(GameObject handle)
-                cornerColliders.Add(cube.GetComponent<Collider>());
-                corners.Add(cube.transform);
+            edgeAxes[1] = CardinalAxisType.X;
+            edgeAxes[3] = CardinalAxisType.X;
+            edgeAxes[5] = CardinalAxisType.X;
+            edgeAxes[7] = CardinalAxisType.X;
 
-                if (handleMaterial != null)
-                {
-                    cubeRenderer.material = handleMaterial;
-                }
-            }
-        }
-
-        private void AddLinks()
-        {
-            edgeCenters = new Vector3[12];
-
-            CalculateEdgeCenters();
-
-            for (int i = 0; i < edgeCenters.Length; ++i)
-            {
-                var ball = GameObject.CreatePrimitive(PrimitiveType.Sphere);
-                ball.name = $"midpoint_{i}";
-                ball.transform.localScale = new Vector3(ballRadius, ballRadius, ballRadius);
-                ball.transform.position = edgeCenters[i];
-                ball.transform.parent = rigRoot.transform;
-
-                var ballRenderer = ball.GetComponent<Renderer>();
-                ballRenderers.Add(ballRenderer);
-                ballColliders.Add(ball.GetComponent<Collider>());
-                balls.Add(ball.transform);
-
-                if (handleMaterial != null)
-                {
-                    ballRenderer.material = handleMaterial;
-                }
-            }
-
-            edgeAxes = new CardinalAxisType[12];
-
-            edgeAxes[0] = CardinalAxisType.X;
-            edgeAxes[1] = CardinalAxisType.Y;
-            edgeAxes[2] = CardinalAxisType.X;
-            edgeAxes[3] = CardinalAxisType.Y;
-            edgeAxes[4] = CardinalAxisType.X;
-            edgeAxes[5] = CardinalAxisType.Y;
-            edgeAxes[6] = CardinalAxisType.X;
-            edgeAxes[7] = CardinalAxisType.Y;
-            edgeAxes[8] = CardinalAxisType.Z;
-            edgeAxes[9] = CardinalAxisType.Z;
-            edgeAxes[10] = CardinalAxisType.Z;
-            edgeAxes[11] = CardinalAxisType.Z;
+            edgeAxes[08] = CardinalAxisType.Y;
+            edgeAxes[09] = CardinalAxisType.Y;
+            edgeAxes[10] = CardinalAxisType.Y;
+            edgeAxes[11] = CardinalAxisType.Y;
 
             for (int i = 0; i < edgeCenters.Length; ++i)
             {
                 var link = GameObject.CreatePrimitive(wireframeShape == WireframeType.Cubic
                     ? PrimitiveType.Cube
                     : PrimitiveType.Cylinder);
-                link.name = $"link_{i}";
 
-                Vector3 linkDimensions = GetLinkDimensions();
+                link.name = $"WireframeEdge_{edgeAxes[i]}_{i}";
+
+                var linkDimensions = GetLinkDimensions();
 
                 switch (edgeAxes[i])
                 {
-                    case CardinalAxisType.Y:
+                    case CardinalAxisType.X:
                         link.transform.localScale = new Vector3(linkRadius, linkDimensions.y, linkRadius);
```

Comment:
Is this correct, the switch for cardinalaxistype X is testing the linkdimensions.Y ?

Reply:
Yes they were not correct before. It was really confusing at first because they didn't quite line up with what I was seeing. Now the physical aligns with the names.